x/feemarket/keeper/keeper_test.go

```diff
@@ -122,20 +127,20 @@ func (suite *KeeperTestSuite) SetupApp(checkTx bool) {
 		CodeHash:    common.BytesToHash(crypto.Keccak256(nil)).String(),
 	}
 
+	acc.AccountNumber = suite.app.AccountKeeper.NextAccountNumber(suite.ctx)
 	suite.app.AccountKeeper.SetAccount(suite.ctx, acc)
 
 	valAddr := sdk.ValAddress(suite.address.Bytes())
-	validator, err := stakingtypes.NewValidator(valAddr, priv.PubKey(), stakingtypes.Description{})
+	validator, err := stakingtypes.NewValidator(valAddr.String(), priv.PubKey(), stakingtypes.Description{})
 	require.NoError(t, err)
-	validator = stakingkeeper.TestingUpdateValidator(suite.app.StakingKeeper, suite.ctx, validator, true)
-	err = suite.app.StakingKeeper.Hooks().AfterValidatorCreated(suite.ctx, validator.GetOperator())
+	err = suite.app.StakingKeeper.SetValidatorByConsAddr(suite.ctx, validator)
```

Comment:
It looks same to below
https://github.com/xpladev/ethermint/pull/14/files#diff-17c4e5810e38949d62886789d63a0de3c1ea03be8dafef51da8e9c6dc95a9302R139

Is there a reason why you removed `TestingUpdateValidator` and the `Hooks`?

Reply:
Is there a reason why you removed `TestingUpdateValidator` and the `Hooks`?

Reply:
https://github.com/xpladev/ethermint/pull/14/commits/1329587c9fd2de40733756cb1f10324574e4d953